online-rest-api/src/main/java/eu/itesla_project/online/rest/model/Indicator.java

```diff
@@ -23,6 +24,12 @@ public Indicator(IndicatorEnum id, UnitEnum unit, double value) {
         this.value = value;
     }
 
+    public Indicator(IndicatorEnum id, float value, LimitViolationType violationType) {
+        this.id = Objects.requireNonNull(id);
+        this.unit = Objects.requireNonNull(violationType).equals(LimitViolationType.CURRENT)?UnitEnum.MW:UnitEnum.KV;
```

Comment:
I think we have to create a specific method to get the unit associated to a LimitViolationType (If we add in a future version the reactive limits (QMIN, QMAX), we'll certainly forget to fix the indicator constructor.